packages/labs/ssr-dom-shim/src/lib/events.ts

```diff
@@ -278,10 +274,9 @@ const EventTargetShim = class EventTarget
     }
     return finishDispatch();
   }
-};
+}
 
-const EventTargetShimWithRealType =
-  EventTargetShim as object as typeof EventTarget;
+const EventTargetShimWithRealType = EventTarget as object as typeof EventTarget;
```

Comment:
What is this doing, exactly? `EventTarget as object as typeof EventTarget`

Reply:
Oh, good catch. That should be `EventTarget as object as typeof globalThis.EventTarget`.

Fixed.